src/components/TouchableOpacity.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * https://github.com/facebook/react-native/blob/master/Libraries/Components/Touchable/TouchableWithoutFeedback.js
+ */
+import React from 'react';
+import EdgeInsetsPropType from '../ropTypes/EdgeInsetsPropType';
+import View from './View';
+// import Touchable from './Touchable';
```

Comment:
Because I overwrote the history to keep it clean. If you go to the "Files Changed" section, it will show the new commit. It is also listed below with id: 0f389ec. :)